cypress/integration/details_spec.js

```diff
@@ -70,4 +68,9 @@ describe('Details page user flow', () => {
         .should('contain', 'Oops, something went wrong')
     })
 
+    it('Should have a specific URL when the user is visiting the details page', () => {
```

Comment:
I'm glad we remembered to test the pathname to ensure router is working as we anticipated!